packages/core/admin/admin/src/components/NpsSurvey/index.js

```diff
@@ -63,29 +101,15 @@ const checkIfShouldShowSurvey = (settings) => {
   return true;
 };
 
-// Exported to make it available during admin user registration.
-// Because we only enable the NPS for users who subscribe to the newsletter when signing up
-export function useNpsSurveySettings() {
-  const [npsSurveySettings, setNpsSurveySettings] = usePersistentState(
-    'STRAPI_NPS_SURVEY_SETTINGS',
-    {
-      enabled: true,
-      lastResponseDate: null,
-      firstDismissalDate: null,
-      lastDismissalDate: null,
-    }
-  );
-
-  return { npsSurveySettings, setNpsSurveySettings };
-}
-
 const NpsSurvey = () => {
+  const { formatMessage } = useIntl();
   const { npsSurveySettings, setNpsSurveySettings } = useNpsSurveySettings();
+  const [feedback, setFeedback] = useState('');
+  const [showFeedbackBox, setShowFeedbackBox] = useState(false);
+  const [selectedRating, setSelectedRating] = useState(null);
```

Comment:
and also this one
`const [selectedRating, setSelectedRating] = React.useState(null);`